testsuite/cluster-test/src/main.rs

```diff
@@ -81,6 +83,8 @@ struct Args {
     perf_run: bool,
     #[structopt(long, group = "action")]
     cleanup: bool,
+    #[structopt(long, group = "action")]
+    changelog: Option<String>,
```

Comment:
Why do we need an `Option<String>` here instead of a `String`? We are not doing anything when args.changelog is None

Reply:
It is `Option<>` because its an action - so it can be present, or not present.
I think Option here is better then comparing to `""`

Reply:
Makes sense. I was confused as to what would happen if we just provide `--changelog` without any String, looks like the structopt library won't allow this.